src/aap_eda/services/activation/tee_log_handler.py

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+from typing import Union
+
+from aap_eda.services.activation.db_log_handler import DBLogger
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TeeLogger(DBLogger):
```

Comment:
Can you add a docstring clarifying the purpose of the class? Maybe use a more self descriptive name like "TeeSystemLogger" too ?